rgdps/models/friend_request.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+from datetime import datetime
+from typing import Any
+from typing import Mapping
+
+
+@dataclass
+class FriendRequest:
+    id: int
+    sender_user_id: int
+    recipient_user_id: int
+    message: str
+    post_ts: datetime
+    seen_ts: datetime | None
+
+    @staticmethod
+    def from_mapping(friend_request_dict: Mapping[str, Any]) -> FriendRequest:
```

Comment:
nit: similar to `UserRelationship` comment

Reply:
Changed in 4e22e55